config/initializers/doorkeeper.rb

```diff
@@ -494,3 +497,6 @@
   #
   # realm "Doorkeeper"
 end
+
+Doorkeeper::OAuth::TokenResponse.send :prepend, Doorkeeper::CustomTokenResponse
+Doorkeeper::OAuth::ErrorResponse.send :prepend, Doorkeeper::CustomErrorResponse
```

Comment:
Next time, do think about links to the docs to document your PR e.g https://github.com/doorkeeper-gem/doorkeeper/wiki/Customizing-token-error-response

It helps reviewers in being confident in the implemented solution.

Reply:
Sure, next time I'll add more info, I thought that a short description is enough